public/app/core/reducers/navModel.ts

```diff
@@ -22,22 +22,24 @@ function buildNavIndex(navIndex: NavIndex, children: NavModelItem[], parentItem?
   }
 }
 
-export const initialState: NavIndex = buildInitialState();
+export const initialState: NavIndex = {};
```

Comment:
Moved this initialization to the rootReducer so that we don't execute functions in the global scope. Please let me know it this causes any unknown issues